packages/nc-gui-v2/assets/style-v2.scss

```diff
@@ -102,7 +102,7 @@ html {
 
 // for highlighting toolbar menu item
 .nc-active-btn > .ant-btn{
-  @apply bg-primary/20;
+  @apply bg-primary/20 hover:(bg-primary/20);
```

Comment:
isn't this redundant?

Reply:
yes otherwise ant-design overriding the style and setting a white background on hover